src/index.js

```diff
@@ -0,0 +1,17 @@
+// import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+// import App from './App';
```

Comment:
- [ ] Please uncomment these two lines because it is throwing an error when I try to run your app